### ISSUE TITLE
Incorrect Dexcom Share server

### ISSUE THREAD

**Issue by unsoluble** [LoopKit/Loop#1401]
Latest Loop is using `shareous2.dexcom.com` for its Outside US server, which is deprecated and does not work. Should be `shareous1.dexcom.com` (see https://github.com/nightscout/share2nightscout-bridge/commit/ea868582d8fb6fcd3be8c6e320283cbc3549eec2).

**Comment by unsoluble** [comment]
Here's the commit to `dev` that swapped to shareous2: https://github.com/LoopKit/dexcom-share-client-swift/commit/68ea5d08588e00bf148518b126416b8352dbef64

**Comment by unsoluble** [comment]
Same thing was done on the Nightscout side of things a while back, still not sure why; we had to revert there as well.

**Comment by PieterGit** [comment]
@unsoluble Proposed a fix in https://github.com/LoopKit/dexcom-share-client-swift/pull/19

**Comment by PieterGit** [comment]
Tested and it works! Tested with:
```
Loop (Automatic Bolusing) v2.3.0
* gitRevision: f875d18ee6be800c7ac8b384837cb9ee1b7d5614
* gitBranch: (HEAD detached at f875d18e)
```

@unsoluble Can you also confirm that this fixes the problem for you?

**Comment by unsoluble** [comment]
Right in the middle of some delicate data collection on the kid's phone for the next few weeks, so don't want to mess with anything right now, but will test when I can. Thanks for writing up the fix! :)

**Comment by Adirael** [comment]
I have the same problem, but I'm not sure if this is working or I'm not compiling it properly. I'm using LoopWorkspace, with the autobolus branch.

This is how I compile: 

```
git clone --branch=automatic-bolus --recurse-submodules https://github.com/LoopKit/LoopWorkspace
git config -f .gitmodules submodule.dexcom-share-client-swift.branch dev
cd dexcom-share-client-swift
git checkout -b dev --track origin/dev
git submodule update --remote
```

The end result is that I can see the change in the ShareClient submodule inside Xcode (ShareClient.swift:32 `case NON_US="https://shareous1.dexcom.com"`) but when I compile and try the change, I still get the error (the app tries to connect with shareous2).

I changed the bundle id to avoid the keychain autofilling but it still tries to connect to the wrong URL. Maybe I'm not compiling it properly so it's not getting changes?

**Comment by TwoFifteen** [comment]
@PieterGit - I see talk here about the automatic-bolus branch but did this fix end up in 2.2.4 master? I'm on the master branch and receiving the same error when trying to add a Dexcom account for Outside US. 

Its referencing shareous2.dexcom.com 

![image](https://user-images.githubusercontent.com/437288/106690866-5f5f3f00-6587-11eb-9ed6-534f5a799c7f.png)
